Umee-Bots/Stale-Price/src/agent.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  BlockEvent,
+  ethers,
+  Finding,
+  getEthersProvider,
+  HandleBlock,
+  HandleTransaction,
+  Initialize,
+  TransactionEvent,
+} from "forta-agent";
+
+import CONFIG from "./agent.config";
+
+import utils, { AgentConfig, AssetDataI } from "./utils";
+
+const assetsDataList: AssetDataI[] = [];
+
+export const provideInitialize =
+  (provider: ethers.providers.Provider): Initialize =>
+  async () => {
+    assetsDataList.push(...(await utils.getAssetData(CONFIG, provider)));
+  };
+
+export const provideHandleTransaction = (
+  config: AgentConfig,
+  provider: ethers.providers.Provider,
+  assetsDataList: AssetDataI[]
+): HandleTransaction => {
+  const handleTransaction: HandleTransaction = async (txEvent: TransactionEvent): Promise<Finding[]> => {
+    const findings: Finding[] = [];
+    const updateSourceLogs = txEvent.filterLog(utils.EVENT_ABI, config.umeeOracleAddress);
+    await Promise.all(
+      updateSourceLogs.map(async (logs) => {
+        const [asset, source] = logs.args;
+        const assetsDataIndex = assetsDataList.findIndex((assetsData) => {
+          return assetsData.asset;
+        });
+        const lastUpdatedAt = await utils.fetchLatestTimestamp(source, provider);
+
+        if (assetsDataIndex === -1) {
+          assetsDataList.push({
+            asset,
```

Comment:
```suggestion
            asset: asset.toLowerCase(),
```
Since this address is going to be used for comparisons, it's good to make sure it's always lowercase because the addresses `0xa` and `0xA` are the same, but the strings `"0xa"` and `"0xA"` are not.